test/end-to-end/multi-supervisor/run_test_case.sh

```diff
@@ -41,11 +49,11 @@ export TESTCASE="${testcase}"
 export TESTCASE_DIR="./testcases/${testcase}"
 
 echo "Validating configuration..."
-docker-compose config
+docker compose config
```

Comment:
I'm seeing failures with `docker compose` see https://buildkite.com/chef/habitat-sh-habitat-main-end-to-end/builds/22#0198e384-235a-4135-bfb0-dc7cd49d390d